src/entities/AccessToken.ts

```diff
@@ -0,0 +1,33 @@
+import moment from "moment";
+
+export interface IAccessToken {
+  access_token: string;
+  expires_in: number;
+  token_type: string;
+  scope: [string];
+  refresh_token: string;
+  patient: string;
+  expires_at?: number;
+}
+
+export default class AccessToken implements IAccessToken {
+  public access_token: string;
```

Comment:
```suggestion
  public accessToken: string;
```

We could then also name these variable in camel case instead. So it'll take "raw json" from the server as part of the constructor and then turn it into a useful class that provides value.

Reply:
@nbragdon I changed this class back to `AuthorizationToken` in the PR and copied the camel case namings from James's PR. TY!